config/config.go

```diff
@@ -428,6 +430,30 @@ type DefReqFiles struct {
 	FileName string `mapstructure:"name"`
 }
 
+type Debug struct {
+	TimeoutNotification TimeoutNotification `mapstructure:"timeout_notification"`
+}
+
+func (cfg *Debug) validate(errs configErrors) configErrors {
+	return cfg.TimeoutNotification.validate(errs)
+}
```

Comment:
Should this function run inside `func (cfg *Configuration) validate() configErrors`? I can't find the place where it gets called.